app-office/onlyoffice-bin/onlyoffice-bin-8.1.0.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop unpacker xdg
+
+MY_P="ONLYOFFICE-DesktopEditors-"${PV}""
+
+DESCRIPTION="Onlyoffice is an office productivity suite (binary version)"
+HOMEPAGE="https://www.onlyoffice.com/"
+SRC_URI="
+	amd64? (
+		https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v"${PV}"/onlyoffice-desktopeditors_amd64.deb
+		-> "${P}"_amd64.deb
+	)
+"
+S="${WORKDIR}"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror strip test"
+
+RDEPEND="
+	>=app-accessibility/at-spi2-core-2.46.0:2
+	dev-libs/expat
+	dev-libs/glib:2
+	dev-libs/nspr
+	dev-libs/nss
+	dev-qt/qtdeclarative:5
+	dev-qt/qtgui:5[eglfs]
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/gst-plugins-base:1.0
+	media-libs/gstreamer:1.0
+	media-libs/libglvnd
+	media-libs/mesa
+	net-print/cups
+	sys-apps/dbus
+	sys-devel/gcc
```

Comment:
Huh, does it really? :D

Reply:
It links against gcc_s.so, so it would be correct imo to include it. A gcc-less system would suffer without it.

Reply:
Ah yes, the llvm profile allows you to avoid it...